src/EnergyPlus/TARCOGArgs.cc

```diff
@@ -425,19 +425,21 @@ int ArgCheck(EnergyPlusData &state,
             return ArgCheck;
         }
 
-        if BITF_TEST_NONE (BITF(LayerType(i)),
+        if
+            BITF_TEST_NONE(BITF(LayerType(i)),
```

Comment:
I :heart: :heart:  the idea of turning the BITF stuff into proper functions.

Reply:
Yes, especially since functions can be made `constexpr`. I've already put something about this on the [wiki](https://github.com/NREL/EnergyPlus/wiki/The-C-Preprocessor)